lib/selinuxtest.pm

```diff
@@ -93,4 +90,58 @@ sub check_category {
     }
 }
 
+sub reboot_and_reconnect {
```

Comment:
ok, good to have a function extracted but why should "selinux" have a special `reboot_and_reconnect` function? I doubt anything needs to be special here compared to other tests. If such functionality is not already in place in a more generic lib file then please move it there, e.g. lib/power_action_utils.pm . For this I recommend to also extend t/04_power_action_utils.t accordingly

Reply:
Understood. But I would like to keep it here.
In this case it sets up selinux envs and does reboot to take effective and I remember the reboot failed on timeout on pvm backend randomly.